manual/src/refman/extensions/alerts.etex

```diff
@@ -17,30 +17,64 @@ component is referenced).
 
 The "ocaml.alert" or "alert" attribute serves two purposes: (i) to
 mark component with an alert to be triggered when the component is
-referenced, and (ii) to control which alert names are enabled.  In the
-first form, the attribute takes an identifier possibly
+referenced, and (ii) to control which alert names are enabled.
+
+\subsection{ss:marking}{Marking component with an alert}
+
+In the first form, the attribute takes an identifier possibly
 followed by a message. Here is an example of a value declaration marked
```

Comment:
`In the first form` sounds even hazier to me once split into a separate section. Should we expand it to

> When a marking  a component, the attribute ...

?